README.md

```diff
@@ -1,28 +1,89 @@
-# Images for Gesso 1.x, F1UX, and F1Omega (Experimental)
+# Images for Gesso 2.x, Gesso 1.x, F1UX, and F1Omega (Experimental)
 
 ## About
 
-This repo builds multiple images for old (pre-`node-sass`) ForumOne themes. Images are broken down by the available Node, PHP, and Ruby versions installed - this is done in case there are version compatibility issues with software.
+This repo builds multiple images for older ForumOne themes. We provide builds for Gesso 2.x and the pre-`node-sass` themes (Gesso 1.x, f1ux, f1omega, and so on). Images are broken down by the available Node, PHP, and Ruby versions installed - this is done in case there are version compatibility issues with software.
```

Comment:
~ForumOne~ Forum One
```suggestion
This repo builds multiple images for older Forum One themes. We provide builds for Gesso 2.x and the pre-`node-sass` themes (Gesso 1.x, f1ux, f1omega, and so on). Images are broken down by the available Node, PHP, and Ruby versions installed - this is done in case there are version compatibility issues with software.
```